src/data/security-quiz.ts

```diff
@@ -8,6 +8,26 @@ const securityQuiz = [
     Explanation: "There is no attack model named CVA (Chosen vector attack).",
     Link: "https://en.wikipedia.org/wiki/Attack_model"
   },
+  {
+      Question:
```

Comment:
it looks like there are some formatting issues

Please run `npm run format` to fix this